sfge/src/main/java/com/salesforce/graph/symbols/apex/ApexValue.java

```diff
@@ -223,8 +223,6 @@ public boolean isNull() {
         if (ValueStatus.UNINITIALIZED.equals(status)) {
             // Uninitialized values are initialized to null by the compiler
             return true;
-        } else if (ValueStatus.INDETERMINANT.equals(status)) {
```

Comment:
This was preventing indeterminants from being properly coerced to `null`, which was causing some of my tests to fail.

Reply:
This is actually an important distinction we make. We don't assume a value is null unless we know for sure. I'm trying to remember better, but Apex's initialization is different from Java's. Could you point out which tests fail? We may need an additional fix.

Reply:
Actually, I can run this in my local as well. Will dig in some more.

Reply:
@rmohan20 , Without this change, all cases of `ApexNullPointerExceptionRuleTest#testNullConstrainedIndeterminant_expectViolation()` fail.